mtlpy/templates/base.html

```diff
@@ -25,7 +25,7 @@
             var js, fjs = d.getElementsByTagName(s)[0];
             if (d.getElementById(id)) return;
             js = d.createElement(s); js.id = id;
-            js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1";
+            js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8";
```

Comment:
Correct answer is : https://wiki.whatwg.org/wiki/HTML_vs._XHTML#Element-specific_parsing

`script and style	In HTML, these are parsed as CDATA elements.`